.golangci.yml

```diff
@@ -53,6 +53,10 @@ issues:
     - linters:
         - revive
       text: "redefines-builtin-id"
+    # FIXME temporarily suppress these. See https://github.com/containerd/containerd/issues/10474
+    - text: "SA1019: grpc.(WithBlock|FailOnNonTempDialError|WithReturnConnectionError|DialContext|Dial) is deprecated"
+      linters:
+        - staticcheck
```

Comment:
I added this exclude-rule for static check. Please see https://github.com/containerd/containerd/issues/10474 for details.